portable/GCC/ARM_CM23_NTZ/non_secure/port.c

```diff
@@ -185,9 +186,11 @@ typedef void ( * portISR_t )( void );
 #define portMPU_MAIR0_REG                       ( *( ( volatile uint32_t * ) 0xe000edc0 ) )
 #define portMPU_MAIR1_REG                       ( *( ( volatile uint32_t * ) 0xe000edc4 ) )
 
-#define portMPU_RBAR_ADDRESS_MASK               ( 0xffffffe0 ) /* Must be 32-byte aligned. */
+#define portMPU_RBAR_ADDRESS_MASK               ( 0xffffff00 ) /* Must be 32-byte aligned. */
```

Comment:
Was this a typo in the original port? Why do we need to now change the RBAR mask? This seems to imply the region is larger than the minimum size - https://developer.arm.com/documentation/dui0662/b/Cortex-M0--Peripherals/Memory-Protection-Unit/MPU-Region-Base-Address-Register?lang=en